Resources/Prototypes/DeltaV/Actions/types.yml

```diff
@@ -1,4 +1,4 @@
-﻿- type: entity
+- type: entity
```

Comment:
untouch

Reply:
how do I do that?

Reply:
i believe this is an encoding change so you can't